src/commands/integration-test-install-tools.yaml

```diff
@@ -5,7 +5,9 @@ steps:
   - run:
       name: "architect/integration-test-install-tools: Install kind"
       command: |
-        CGO_ENABLED=0 GO111MODULE="on" go get sigs.k8s.io/kind@v0.7.0
+        curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.7.0/kind-$(uname)-amd64"
+        chmod +x ./kind
+        sudo mv ./kind /usr/local/bin
```

Comment:
Nice catch, thx! 

IMO safer to use the binary to remove the risk of affecting go modules.